test/system/tests/nv.sh

```diff
@@ -58,26 +60,6 @@ onerror() {
 }
 trap onerror ERR
 
-function yaml_get() {
```

Comment:
This should also be in a separate patch. Basically I would do:

1) Factor out all the existing funcions in helper.sh as the first patch
2) Add any new functions that are going to be used
3) Do all the YAML conversions and sourcing helpers.sh in the tests.